### DISCUSSION TITLE
Support request for devices behind a gateway that do not update their status

### DISCUSSION THREAD

**Discussion by Roby2002** [make-all/tuya-local#2206] [Q&A]
I have a "diivoo dual water" valve, unfortunately I can't keep the gateway always on, so I activate it only when I need it, but in home assistant the entities don't update and remain offline. To see them I have to change the valve status manually or restart home assistant.
Is there any command to be able to "force" the alignment after I have turned on the gateway? Other solutions?

**Comment by make-all** [comment]
Did you set the IP address and protocol version for all your devices?

**Reply by Roby2002** [reply]
For this valve yes, I set the IP and protocol. 
I have other devices, where I set the IP and left the protocol in auto, is that not ok?
